docs/iris/src/developers_guide/release.rst

```diff
@@ -1,75 +1,172 @@
 .. _iris_development_releases:
 
 Releases
-********
+========
+
+A release of Iris is a `tag on the SciTools/Iris`_ 
+Github repository.
+
+The summary below is of the main areas that constitute the release.  The final
+section details the :ref:`iris_development_releases_steps` to take.
 
-A release of Iris is a tag on the SciTools/Iris Github repository.
 
 Release branch
-==============
+--------------
 
-Once the features intended for the release are on master, a release branch should be created, in the SciTools/Iris repository.  This will have the name:
+Once the features intended for the release are on master, a release branch 
+should be created, in the SciTools/Iris repository.  This will have the name:
 
     :literal:`{major release number}.{minor release number}.x`
 
 for example:
 
     :literal:`v1.9.x`
 
-This branch shall be used to finalise the release details in preparation for the release candidate.
+This branch shall be used to finalise the release details in preparation for
+the release candidate.
+
 
 Release candidate
-=================
+-----------------
 
-Prior to a release, a release candidate tag may be created, marked as a pre-release in github, with a tag ending with :literal:`rc` followed by a number, e.g.:
+Prior to a release, a release candidate tag may be created, marked as a
+pre-release in github, with a tag ending with :literal:`rc` followed by a
+number, e.g.:
 
     :literal:`v1.9.0rc1`
 
-If created, the pre-release shall be available for at least one week prior to the release being cut.  User groups should be notified of the existence of the pre-release and encouraged to test the functionality.
+If created, the pre-release shall be available for at least one week prior to
+the release being cut.  User groups should be notified of the existence of the
+pre-release and encouraged to test the functionality.
 
-A pre-release is expected for a minor release, but not normally provided for a point release.
+A pre-release is expected for a minor release, but not normally provided for a
+point release.
```

Comment:
@tkknight I'd be firmer than that, and state that there will not be any `rc`s for a point/patch release

Reply:
That's actually stated later on... see [here](https://github.com/SciTools/iris/pull/3769/files#diff-357e85c5f819fb7f0efb53a02eeee65bR113)